docs/v2-system-addon/vendor.md

```diff
@@ -0,0 +1,44 @@
+# Vendoring in npm libraries
+
+#### Build system basics
+
+#### Vendored libraries as externals
+
+#### Is this library a good candidate to depend on and vendor?
+
+#### How to vendor in a library
+
+- [ ] file a bug for vendoring in the library itself
+
+- [ ] Get license approval from the licensing team
+
+  - [ ] review the mozilla-central policy at
+        https://www.mozilla.org/en-US/MPL/license-policy/
+  - [ ] file a license bug as described in the policy doc to request approval
```

Comment:
License approval bug for BSD 3-Clause, https://bugzilla.mozilla.org/show_bug.cgi?id=1570697